engine/apps/integrations/mixins/ratelimit_mixin.py

```diff
@@ -6,9 +6,9 @@
 from django.core.cache import cache
 from django.http import HttpRequest, HttpResponse
 from django.views import View
-from ratelimit import ALL
-from ratelimit.exceptions import Ratelimited
-from ratelimit.utils import is_ratelimited
+from django_ratelimit import ALL
+from django_ratelimit.exceptions import Ratelimited
+from django_ratelimit.utils import is_ratelimited
```

Comment:
https://django-ratelimit.readthedocs.io/en/stable/upgrading.html#from-2-0-to-3-0
https://django-ratelimit.readthedocs.io/en/stable/upgrading.html#from-3-x-to-4-0